### ISSUE TITLE
sqlite?

### ISSUE THREAD

**Issue by ghenry** [librestack/libgladdb#7]
Hi Brett,

I'm looking to use this for https://github.com/SentryPeer/SentryPeer

Worth me adding sqlite support too?

Thanks.

**Comment by brettsheffield** [comment]
Cool & thanks - go for it.  Just send me a PR when you're ready.

**Comment by brettsheffield** [comment]
NB: Planning to relicence this as GPL 2 + GPL 3, but that seems to match your project so that won't be a problem.

**Comment by ghenry** [comment]
Great! Saves me looking at other things later. I just referenced a lot of the librecast project layout. Been reading a lot of the 0mq guides and docs. Such a shame he's left us. Ever played with zproject for a lib? gsl etc. all look good but worry about the life of them now Pieter is gone. But it's been 5 years and still looks strong, the whole ZeroMQ ecosystem.

Was looking at Zyre for SentryPeer backend.

**Comment by brettsheffield** [comment]
Pieter made sure these projects would outlive him. He had a lot of time to prepare and the community around them is pretty strong.

I don't use zyre or zproject (or zeromq for that matter). Everything we do is based on Librecast.

**Comment by ghenry** [comment]
Yep, I read that. Did you ever meet him?

OK, good to know. Something I want to try anyway. librecast in my project.

**Comment by brettsheffield** [comment]
No, never met him. He died shortly before I started getting interested in his work.

If you do start looking at librecast, let me know if you need any help. The library is available in Debian Experimental, but there are a lot of other bits that might be useful that I'm working on, such as authentication.

**Comment by ghenry** [comment]
Shame.

I will be later. Will let you know.
